ebos/ecltracermodel.hh

```diff
@@ -106,9 +107,10 @@ public:
 
         if (!EWOMS_GET_PARAM(TypeTag, bool, EnableTracerModel)) {
             if (simulator_.gridView().comm().rank() == 0) {
-                std::cout << "Warning: Tracer model is disabled but the deck contains the TRACERS keyword\n"
-                          << "The tracer model must be explictly activated using --enable-tracer-model=true\n"
-                          << std::flush;
+                OpmLog::warning("Warning: Keyword TRACERS has only experimental support, and is hence ignored.\n"
```

Comment:
The logging system will add the `Warning:` prefix - so this will be `Warning: Warning:` when it hits the log file

Reply:
Bah, you are completely right, it was a left over from the message it replaced. Will fix.